low-level/framework/sspl_ll_d

```diff
@@ -819,7 +819,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
-        logger.debug(f"{traceback.print_exc()}")
+        logger.exception("Error while initiliazing sspl-ll service : ")
         logger.critical("While spawning sspl-ll process :%r" % err)
```

Comment:
We can remove the critical logging as we are already logging it with logger.exeption.

Reply:
Done.